charts/greenhouse/templates/pluginconfig-kube-monitoring.yaml

```
@@ -20,6 +28,11 @@ spec:
       cluster: {{ .Values.monitoring.extLabels.cluster }}
       region: {{ .Values.monitoring.extLabels.region }}
 {{- end }}
+  - name: kubeMonitoring.prometheus.tlsConfig.caCert
+    valueFrom:
+      secret:
+        key: "ca.crt"
+        name: kube-monitoring-ingress-ca
```

This option is no longer used. I neglected to remove this option from the readme file. 
```suggestion
  - name: kubeMonitoring.prometheus.annotations.nginx\.ingress\.kubernetes\.io/auth-tls-secret
    value: greenhouse/kube-monitoring-ingress-ca
```

---

Thanks :) 
Added this as another example for our discussion w.r.t. to rendering hard to read `PluginOptions` on the PluginAdmin UI. Not only deeply nested but also requires the additional escaping.